src/main/java/domain/ProposalStatus.java

```diff
@@ -1,33 +1,123 @@
 package domain;
 
+import org.hibernate.validator.constraints.NotBlank;
+import javax.persistence.*;
 import java.util.Calendar;
 
 public class ProposalStatus
 {
-    public ProposalStatus() {}
+    @Column
+    @Id
+    @GeneratedValue
+    private Integer id;
+
+    @Column
+    @NotBlank(message="proposalStatus.status")
+    private String status;
+
+
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column
+    private Calendar modified;
+
+
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column
+    private Calendar created;
+
+    @Column
+    @NotBlank(message="proposalStatus.comment")
+    private String comment;
+
+    public ProposalStatus(){
+        this(null,null,null,null);
+    }
+
+    public ProposalStatus(String status, Calendar modified, Calendar created, String comment) {
```

Comment:
Si vezi ca asta trebuie sa fie legata ManyToOne la User si Proposal, se vede din UML, alea 2 relatii apar si aici doar ca nu apar si atributele lor in schema.